wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/tool/dockerimage/DockerimageToolToWorkspaceApplier.java

```diff
@@ -133,4 +144,26 @@ public void apply(
                     .equals(c.getAttributes().get(Constants.TOOL_NAME_COMMAND_ATTRIBUTE)))
         .forEach(c -> c.getAttributes().put(MACHINE_NAME_ATTRIBUTE, machineName));
   }
+
+  @SuppressWarnings("unchecked")
```

Comment:
There is no way to cast `Object` into `List<String>` without unchecked cast warnings that I know of. That said, this will disappear because we're dropping the support for string values and will only keep list of string to be inline with kubernetes,